docs/install.md

```diff
@@ -276,60 +89,74 @@ This is borrowed from the [NixOS manual](https://nixos.org/nixos/manual/index.ht
     nano /mnt/etc/nixos/configuration.nix
     ```
 
-    Option 1: Edit NixOS configuration for UEFI
+    We now need to adjust the configuration to make sure that we can ssh into the system and that it boots correctly. We add some lines to set `services.openssh` such that the configuration looks as follows:
```

Comment:
Ideally, a public key for ssh should be set here already (when setting up the root login before we deploy nix-bitcoin).  `krops-deploy` asks for the root password for each operation (easily 15-20).

Reply:
Hmm, I don't remember this. Will need to reproduce it. Not so easy to type your ssh key.